examples/swift/SamplePackage/Package.swift

```diff
@@ -0,0 +1,29 @@
+// swift-tools-version: 5.8
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "SamplePackage",
+    products: [
+        // Products define the executables and libraries a package produces, making them visible to other packages.
+        .library(
+            name: "SamplePackage",
+            targets: ["SamplePackage"]),
+    ],
+    dependencies: [
+        .package(path: "../../../wrappers/swift"),
```

Comment:
In our logging library, it would therefore be sufficient to checkout the source (in say a submodule) and point this to the correct location, right?